test/helpers.go

```diff
@@ -51,3 +53,24 @@ func InitConfigForIntegrationsTesting() {
 	// Load normal config from environment variables and config files
 	cmdutil.InitConfig()
 }
+
+func MockEventContainerV2(key string) eventsapi.EventContainer {
```

Comment:
nit: We may want to call this something like `BuildV2EventContainer` or something since it isn't actually a mock...

Reply:
Good point, fixed here: c5ce9ed